labelImg.py

```diff
@@ -937,7 +944,7 @@ def new_shape(self):
                 text = self.label_dialog.pop_up(text=self.prev_label_text)
                 self.lastLabel = text
         else:
-            text = self.default_label_text_line.text()
+            text = self.default_label
```

Comment:
I forgot the difference between  self.default_label_text_line.text()   and self.default_label? Could you elaborate it?

Reply:
Earlier, if default label checkbox was ticked, the class name was extracted from the text-box, this is self.default_label_text_line.text(). 

Now, that it has been converted to a drop-down list, I am setting the value of self.default_label, whenever it is changed. On application startup, it is initialized with the first class from predefined_classes.txt

self.default_label was not present earlier, its a class attribute I have introduced. its only purpose is to hold the selection of the default label drop down.